packages/suite/src/views/onboarding/unexpected-states/components/IsSameDevice.tsx

```diff
@@ -24,33 +19,27 @@ const mapDispatchToProps = (dispatch: Dispatch) =>
 type Props = ReturnType<typeof mapDispatchToProps>;
 
 const IsSameDevice = ({ resetOnboarding, enableOnboardingReducer }: Props) => (
-    <Wrapper.Step>
-        <Wrapper.StepBody>
-            <StyledImage image="UNI_WARNING" />
-
-            <Wrapper.StepHeading>
-                <Translation id="ONBOARDING_UNEXPECTED_DEVICE_DIFFERENT_HEADING" />
-            </Wrapper.StepHeading>
-
-            <Text>
+    <OnboardingStepBox
+        disableConfirmWrapper
+        heading={<Translation id="ONBOARDING_UNEXPECTED_DEVICE_DIFFERENT_HEADING" />}
+        description={
+            <>
                 <Translation id="ONBOARDING_UNEXPECTED_DEVICE_DIFFERENT_P1" />
-            </Text>
-            <Text>
                 <Translation id="ONBOARDING_UNEXPECTED_DEVICE_DIFFERENT_P2" />
-            </Text>
-            <Wrapper.Controls>
-                <OnboardingButton.Alt
-                    onClick={() => {
-                        resetOnboarding();
-                        enableOnboardingReducer(true);
-                    }}
-                    data-test="@onboarding/unexpected-state/is-same/start-over-button"
-                >
-                    <Translation id="TR_START_AGAIN" />
-                </OnboardingButton.Alt>
-            </Wrapper.Controls>
-        </Wrapper.StepBody>
-    </Wrapper.Step>
+            </>
+        }
+        innerActions={
+            <OnboardingButtonAlt
+                onClick={() => {
+                    resetOnboarding();
+                    enableOnboardingReducer(true);
+                }}
+                data-test="@onboarding/unexpected-state/is-same/start-over-button"
+            >
+                <Translation id="TR_START_AGAIN" />
+            </OnboardingButtonAlt>
+        }
+    />
 );
 
 export default connect(null, mapDispatchToProps)(IsSameDevice);
```

Comment:
Would be cool to update this to use the hook instead (maybe in the follow up PR or as a totally separate task)